### ISSUE TITLE
Stickies superposes when resizing & element disappear when detached

### ISSUE THREAD

**Issue by simonvart** [leafo/sticky-kit#22]
Hi. I think I may have find a bug (or my implementation is incorrect).

I have a container, with a floated element that is my sticky element. I wait for the images to load (with imagesLoaded plugin) and then I attach the element with sticky_kit.

On scroll, I load another container with ajax. I apply the same process (wait for the images to load, then attach the element). To avoid detach/attach operation, I check all element with jQuery( this ).data("sticky_kit") and attach only the newly loaded element.

On window resize, beyond a certain window width, I detach all elements. 

Everything works fine, but the problems I saw are : 

-+ When I detach elements, they disappear. In the DOM, only remains a empty div (added by the sticky_kit plugin).

-+ When I resize the browser, the stickies elements get superposed to each other. If I scroll to the top then down to the bottom, each element finally gets back to the right places (and sticks as it should).

-+ I tried to do a recalc on the resize, but I guess the superposition is because of the recalc.

I can provide a link to a live website to see by yourself. You could load the page, resize it (you will see the superposition) and if you resize under 640px, the detach fonction is triggered and the elements disappear.

(Edit : I commented the line 182 win.on("resize", recalc_and_tick); and all seems to work again. Maybe that was because I triggered on window resize and the plugin too : collision ?
